.github/workflows/test-code.yml

```diff
@@ -31,9 +31,8 @@ jobs:
           - ubuntu-latest
           - windows-latest
         graphqlversion:
-          - 5.1.1
-          - 5.2.0
-          - 5.3.1
+          - 7.0.2
```

Comment:
If the base version required by the relay project is 7.1.1 then we don’t need to test against 7.0.2. The testing is to test forward. For example, if the relay project required 5.0+ but was compatible with 7.1.1, we could test for all the versions in between. 

Fyi we usually try to keep projects compatible with as wide array of versions as is practical or makes sense.

Reply:
Ah.  Ok.  Would it make sense to downgrade the GraphQL version to 7.0.2 then?

Reply:
Probably.

Technically it might be compatible with 5.x also (I see no code changes to the main project) — just the tests probably are not cross version compatible. 

I’m fine to use 7.0.2